src/toast/widgets.py

```diff
@@ -272,7 +272,7 @@ def response(change):
             )
             with w_plot:
                 clear_output(wait=True)
-                fig.show_dash(mode="inline", host="localhost")
+                fig.show_dash(mode="inline_persistent", host="localhost")
```

Comment:
This does not seem related but that is fine.

Reply:
yes, a small tweak to the interactive display widget, and unrelated.